src/gnatss/main.py

```diff
@@ -754,6 +754,7 @@ def main(
     config: Configuration,
     all_files_dict: Dict[str, Any],
     extract_process_dataset: bool = False,
+    outlier_threshold: float = 25.0,
```

Comment:
Can this be put in a shared constant so that it's not duplicated in the CLI? To avoid the two values from diverging later on.